contracts/clients/test/index.test.ts

```diff
@@ -57,8 +55,8 @@ describe("index", () => {
     const bundle = sign(bundleTemplate, privateKey);
 
     expect(bundle.signature).to.deep.equal([
-      "0x0058b38298f3c486223de7c61f461ff3b47530d2619a383e49b298a56249e4fb",
-      "0x0bf8beb03979073e57656af0a5bab043fe2d6bf4cbbf600f6a7190ce95fcf69c",
+      "0x0f8af80a400b731f4f2ddcd29816f296cca75e34816d466512a703631de3bb69",
+      "0x023d76b485531a8dbc087b2d6f25563ad7f6d81d25f5f123186d0ec26da5e2d0",
```

Comment:
`mcl-wasm@1.0.0` generates new hexes in these tests. They are still valid in contract and aggregator tests.

Reply:
Weird! But ok :).